src/libaktualizr/http/httpclient.cc

```diff
@@ -320,13 +320,19 @@ std::future<HttpResponse> HttpClient::downloadAsync(const std::string& url, curl
 
 bool HttpClient::updateHeader(const std::string& name, const std::string& value) {
   curl_slist* item = headers;
-  std::string lookfor(name + ": ");
+  std::string lookfor(name);
```

Comment:
This might introduce an issue if some header name is a substring of another.
If we have the headers: `{"my_header_x:1", "my_header:2" }` , `updateHeader("my_header", " 3")` may lead to `{"my_header:3", "my_header:2" }` instead of `{"my_header_x:1", "my_header:3" }`

Reply:
Fixed.